gdb-server/src/worker.rs

```diff
@@ -52,11 +52,20 @@ pub async fn handler(
     let mut break_due = false;
     if packet.is_valid() {
         let packet_string = String::from_utf8_lossy(&packet.data).to_string();
+
         #[allow(clippy::if_same_then_else)]
         let response: Option<String> = if packet.data.starts_with(b"qSupported") {
             handlers::q_supported()
         } else if packet.data.starts_with(b"vMustReplyEmpty") {
             handlers::reply_empty()
+        } else if packet.data.starts_with(b"vCont?") {
```

Comment:
Why did you relocate those handlers? Did you feel like it or is there anything more to it? Either way, fine with me, just curious :)

Reply:
Just wanted to have all the handlers starting with v at the same location.